infra/terraform.tfstate

```diff
@@ -0,0 +1,48 @@
+{
+    "version": 3,
+    "terraform_version": "0.9.3",
+    "serial": 7,
+    "lineage": "a9fd0174-2702-4905-8e58-900013b8496a",
+    "modules": [
+        {
+            "path": [
+                "root"
+            ],
+            "outputs": {},
+            "resources": {
+                "digitalocean_droplet.delta_droplet": {
+                    "type": "digitalocean_droplet",
+                    "depends_on": [],
+                    "primary": {
+                        "id": "45902590",
+                        "attributes": {
+                            "backups": "true",
+                            "disk": "60",
+                            "id": "45902590",
+                            "image": "ubuntu-16-10-x64",
+                            "ipv4_address": "67.205.157.40",
+                            "ipv4_address_private": "10.136.36.61",
+                            "locked": "false",
+                            "name": "delta_lookup_tool",
+                            "private_networking": "true",
+                            "region": "nyc1",
+                            "resize_disk": "true",
+                            "size": "4gb",
+                            "ssh_keys.#": "1",
+                            "ssh_keys.0": "8187378",
+                            "status": "active",
+                            "tags.#": "0",
+                            "user_data": "#cloud-config\n\npackage_upgrade: true\n\npackages:\n  - apt-transport-https\n  - ca-certificates\n  - curl\n\nruncmd:\n  # Install docker-ce\n  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -\n  - sudo add-apt-repository \"deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable\"\n  - sudo apt-get update \u0026\u0026 apt-get install -y docker-ce\n  - sudo curl -o /usr/local/bin/docker-compose -L \"https://github.com/docker/compose/releases/download/1.11.2/docker-compose-$(uname -s)-$(uname -m)\"\n  - sudo chmod +x /usr/local/bin/docker-compose\n\n  # Install the DO monitoring agent\n  - curl -sSL https://agent.digitalocean.com/install.sh | sh\n\n\n  # Get the app up and running\n  - [mkdir, -p, /opt/apps/deltanyc-heatseek]\n  - git clone https://github.com/chrishenry/deltanyc-heatseek /opt/apps/deltanyc-heatseek\n  - cp /opt/apps/deltanyc-heatseek/.env.example /opt/apps/deltanyc-heatseek/.env\n  - cd /opt/apps/deltanyc-heatseek \u0026\u0026 git checkout infra\n  - cd /opt/apps/deltanyc-heatseek \u0026\u0026 docker-compose up -d\n  - cd /opt/apps/deltanyc-heatseek \u0026\u0026 docker-compose exec nb \"for f in `ls | grep import | grep .py | grep -v pyc`; do python $f; done\"\n\n",
```

Comment:
Is this setting a bash script to run on the droplet via user_data? This seems kinda wacky. Is it at least possible to add \ns?